### PR TITLE
Page Developed.

### PR DESCRIPTION
**Pull request by davitomix**
This project consists on create a webpage for a restaurant using JS, webpack, and Classes.

- Project created following the module pattern.
- Used classes and objects for the page functionality.
- Fixed lint issues on the src/ folder.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <nav>
+        <div id="menu">
+          <div id="about" class="item">About</div>
+          <div id="dishes" class="item">Menu</div>
+          <div id="contact" class="item">Contact</div>
+        </div>
+      </nav>
+    </div>
+    <main id="content">
+    </main>
```
Comment (bellom) [inline dist/index.html:20]: You are missing one of the hard requirements of the projects which is using a div in your `index.html` 
Example below;
```
 <body>
   <div id="content"></div>
 </body>
```

From `Odin` Page:
image

Comment (davitomix) [reply dist/index.html]: Hi, the requested change was added.
Thanks.

**Comment by davitomix** [comment]
The changes requested by the previous TSE have been implemented, thanks.

File: dist/index.html (hunk 1 above)
Comment (bellom) [reply dist/index.html]: @davitomix,
Please what this means is; you will only have `div` inside the `body` tag. Then use JS to generate all tags needed.
Just as this 👇 
```
 <body>
   <div id="content"></div>
 </body>
```

Comment (davitomix) [reply dist/index.html]: Got it. Thanks

File: dist/index.html [hunk 2]
```diff
@@ -3,19 +3,7 @@
...
-      </h1>
-      <nav>
-        <div id="menu">
-          <div id="about" class="item">About</div>
-          <div id="dishes" class="item">Menu</div>
-          <div id="contact" class="item">Contact</div>
-        </div>
-      </nav>
-    </div>
+  <body id="body">
```
Comment (davitomix) [inline dist/index.html:6]: - Removed navigation elements as required on project specifications.
- Implemented a Class to inject the navbar on the document.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "RestaurantPageJS",
+  "version": "1.0.0",
+  "description": "> One paragraph statement about the project.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (bellom) [inline package.json:7]: kindly update your script before merging:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Pull request merged**